src/identity/components/userprofile/UserDefaults.tsx

```diff
@@ -74,11 +76,11 @@ export const UserDefaults: FC = () => {
 
   useEffect(() => {
     setSelectedAccount(defaultAccount)
-  }, [accounts, defaultAccount])
+  }, [defaultAccount])
 
   useEffect(() => {
     setSelectedOrg(defaultOrg)
-  }, [orgs, defaultOrg])
+  }, [defaultOrg])
 
```

Comment:
This dependency is not necessary (and causes the selected dropdown item to toggle back to the default if there is an error and a re-render).